remme/shared/forms/_validators.py

```diff
@@ -47,6 +47,32 @@ def __call__(self, form, field):
             raise validators.StopValidation(message)
 
 
+class BooleanTypeRequired(object):
```

Comment:
@Art17, remove this class, because parameter `reverse` is a string type.

Reply:
Done.